src/main/java/io/stargate/sgv2/jsonapi/StargateJsonApi.java

```diff
@@ -562,6 +572,17 @@
                                   }
                                 }
                                 """),
+              @ExampleObject(
+                  name = "resultFindCollections",
+                  summary = "`findCollections` command result",
+                  value =
+                      """
+                                {
+                                  "status": {
+                                      "existingCollections": [ "events" ]
```

Comment:
Minor detail: 'existingCollections' is a bit verbose, maybe just 'collections'? Very minor detail.

Reply:
Done.